R/as_list.R

```diff
@@ -27,6 +27,11 @@ as_list <- function(x, ns = character(), ...) {
 }
 
 #' @export
+as_list.default <- function(x, ns = character(), ...) {
```

Comment:
How much extra would it be to make this work with a specific class? I worry a little that a default method will make it easy for accidental mistakes to silently propagate